changelogs/21.10

```diff
@@ -0,0 +1,72 @@
+* COSMIC
+** GNOME 40/41 support for COSMIC and Pop Shell.
+** Re-enabling support for vertical workspaces.
+** Added a new Application Library interface to replace the GNOME application overview.
+** Various UX improvements to the launcher have been made across this release.
+** And a handful of issues in Pop Shell have been resolved as well.
+** Emoji picker set back to Ctrl + Shift + E
++* Alt+F4 added as an alternative to Super+Q for closing windows
+
+* Application Library
+** Replaces the GNOME application overview, achieving the same information density with less space.
+** Improves multi-monitor experience by launching onto the display that has mouse focus.
+** Alphabetical application sorting will make it easier to locate the application or folder you want.
+** Drag and drop applications into custom folders which are visible and act as tabs.
+** Search quickly filters installed applications and applications that are available to install from the Pop!_Shop.
+** Achieves the same information density while using much less space.
+** Provides the optimal experience, especially on ultra-wide monitors.
+
+* Pop! Shell
+** Various scenarios causing crashes were resolved.
+** Allow floating window exceptions to be tiled when toggled.
+** Transitions removed on opening a window to fix windows being stretched sometimes.
+** Dialogs will no longer unmaximize a maximized window when they gain focus.
+** Improved high contrast support.
+** Many improvements to the tiling behaviors of many unique window types.
+** Improvements to window attachment behaviors when new windows are spawned.
+
+* Pop! Shop
+** Many PRs have been submitted to elementary OS to address a number of problematic areas we've experienced.
```

Comment:
May want to reference "upstream" so someone not familiar with where the Pop!_Shop comes from will know why we're talking about elementary.

> ** Submitted several PRs upstream to the elementary OS project addressing problematic areas.

(If we could get more specifics than "problematic areas" from @isantop or anyone else, that would be good too.)